tests/core/test_chunked_prefill_scheduler.py

```diff
@@ -180,7 +180,7 @@ def test_maximal_decoding():
     """Verify decoding requests are prioritized."""
     block_size = 4
     max_seqs = 2
-    max_model_len = 2
+    max_model_len = 8
```

Comment:
We needed to add a detection into the scheduler of when the sequence reaches max_model_len, since this is the only case of stopping that needs to be detected immediately (and cannot have the extra +1 decode run). As a result, this test started to fail because it was limited to tiny length, where the actual length used is larger.